Resources/public/js/cropper.js

```diff
@@ -1,22 +1,23 @@
+const bootstrap = require('bootstrap');
 const CropperJS = require('cropperjs');
 
 (function(w, $) {
 
     'use strict';
 
-    const Cropper = function($el) {
+    const Cropper = function($el, modalV5 = false) {
         this.$el = $el;
         this.options = $.extend({}, $el.data('cropper-options'));
 
         this
-            .initElements()
+            .initElements(modalV5)
             .initLocalEvents()
             .initRemoteEvents()
             .initCroppingEvents()
         ;
     };
 
-    Cropper.prototype.initElements = function() {
+    Cropper.prototype.initElements = function(modalV5) {
```

Comment:
When possible I prefer avoiding such flag parameter, maybe we could have a version parameter, or different cropper for each version ?

Reply:
I'm working on a new version to remove entirely the bootstrap dependency. This is just a "quickfix" until the new version is released.